src/test-data/ModificationsData.js

```diff
@@ -49,3 +49,40 @@ export const exampleModificationAltersTopLevel = [
     ],
   },
 ];
+
+export const profileModifyTestData = {
```

Comment:
This doesn't seem significantly different from the existing `exampleModifcationAlters` and `exampleModificationSetParameters` objects in this file. Is it possible to modify this (or modify the tests) to use the existing data?

Reply:
If this seems out of the scope of this PR though this doesn't need to hold up approval.

Reply:
Oh I had missed that! Good catch @hreineck. Should be good now.

Reply:
I'm looking at the code and `exampleModificationAltersTopLevel` is almost the exact same as `exampleModificationAlters`. Would it make more sense to just use one of them, so as to get rid of the data duplication?

Reply:
@mikeisen1 I believe that the difference between a `ModificationAlter` & a `ModificationAltersTopLevel` might be significant enough for the duplication. The `by-id` is different for both and I think the amount of code redundancy fixed is very minimal.